appendices/VK_INTEL_performance_query.txt

```diff
@@ -76,7 +76,7 @@ None.
 // A previously created device
 VkDevice device;
 
-// A queue from from device
+// A queue from device
```

Comment:
From *the* device? Really not sure  what the original author intended to say.

Reply:
He is predeclaring boinlerplate with pseudocode. Likely what he means is "a queue created from the `device`".

Reply:
Clearly I was also tired. Which of the two suggestions do you wish me to apply?

Reply:
It was meant to say : queue derived from the device above indeed.

If you can add that that's great, otherwise the current change doesn't make it worse than it was :)

Reply:
Changed it to "A queue queue derived from the device" :)

Reply:
Ha. Copy paste mistake, even in the commit.